packages/luigis-box/src/Model/Product/Filter/LuigisBoxFacetsToProductFilterOptionsMapper.php

```diff
@@ -216,4 +218,14 @@ protected function mapParameterWithValues(
 
         return new ParameterFilterChoice($parameter, $parameterValues);
     }
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Product\Filter\ParameterFilterChoice[] $parameterFilterChoices
+     */
+    protected function orderParameterChoicesByParameterOrderingPriority(array &$parameterFilterChoices): void
+    {
+        usort($parameterFilterChoices, static function (ParameterFilterChoice $a, ParameterFilterChoice $b) {
+            return $b->getParameter()->getOrderingPriority() - $a->getParameter()->getOrderingPriority();
```

Comment:
what if ordering priority is the same? how the parameters should be sorted in that case?